docs/business.yaml

```diff
@@ -839,46 +844,41 @@ paths:
-                    rootCause: errors:[error:Alteration of Name Request has a different legal name.,path:/filing/alteration/nameRequest/legalName]
-                alteration-failed-name-request-is-not-approved-response:
-                  summary: Alteration - Name Request Is Not Approved Response
+                    rootCause: errors:[error:Invalid country.,path:/filing/consentContinuationOut/foreignJurisdiction/country],filing:business:foundingDate:2024-07-08T15:34:57.844764+00:00,identifier:BC0882848,legalName:0882848 B.C. LTD.,legalType:BEN,consentContinuationOut:courtOrder:effectOfOrder:planOfArrangement,fileNumber:12345,foreignJurisdiction:country:,header:availableOnPaperOnly:false,certifiedBy:Api specs,date:2024-07-10,documentOptionalEmail:Apispecs@email.com,email:Apispecs@gov.bc.ca,inColinOnly:false,name:consentContinuationOut
+                consent-continuation-out-failed-invalid-foreign-jurisdiction-region-response:
+                  summary: Consent Continuation Out - Invalid Foreign Jurisdiction Region Response
                   value:
                     errorMessage: API backend third party service error.
-                    rootCause: errors:[error:Alteration of Name Request is not approved.,path:/filing/alteration/nameRequest/nrNumber,error:Alteration of Name Request has a different legal name.,path:/filing/alteration/nameRequest/legalName]
-                alteration-failed-invalid-legal-type-change-response:
-                  summary: Alteration - Invalid Legal Type Change Response
+                    rootCause: errors:[error:Invalid region.,path:/filing/consentContinuationOut/foreignJurisdiction/region],filing:business:foundingDate:2024-07-08T15:34:57.844764+00:00,identifier:BC0882848,legalName:0882848 B.C. LTD.,legalType:BEN,state:HISTORICAL,consentContinuationOut:courtOrder:effectOfOrder:planOfArrangement,fileNumber:12345,foreignJurisdiction:country:CA,region:,header:availableOnPaperOnly:false,certifiedBy:Api specs,date:2024-07-10,documentOptionalEmail:Apispecs@email.com,email:Apispecs@gov.bc.ca,inColinOnly:false,name:consentContinuationOut
+                consent-continuation-out-failed-missing-required-field-response:
+                  summary: Consent Continuation Out - Missing Required Field Response
                   value:
                     errorMessage: API backend third party service error.
-                    rootCause: errors:[error:Your business type has not been updated to a BC Benefit Company, BC Unlimited Liability Company, BC Community Contribution Company, BC Limited Company or BC Cooperative Association.,path:/filing/alteration/business/legalType]
-                alteration-failed-missing-max-number-of-share-response:
-                  summary: Alteration - Missing Max Number of Shares Response
+                    rootCause: errors:[message:A valid business is required.]
+                consent-continuation-out-failed-invalid-jurisdiction-response:
```

Comment:
consider putting this one before `consent-continuation-out-failed-missing-required-field-response`, since `i` is in front of `m`

Reply:
Thanks for the suggestion.

I'm sorting these based on the alphabetical order of the `summary` field because Apigee displays the content based on this field. 

The current sorting results are as follows:
<img width="606" alt="image" src="https://github.com/user-attachments/assets/e1295ed8-56cd-45d7-ae48-61a8deadc08e">

Reply:
Ah, I see, you are sorting by the display name (the summary). Looking good for me.